# 包含一帧图像中所有检测结果的数组
std_msgs/Header header  # 消息头，包含时间戳和坐标系信息
Detection[] detections # 检测结果数组
